src/commands/apm_cmds/publish.js

```diff
@@ -74,6 +76,9 @@ exports.builder = function (yargs) {
     }).option('http-served-from', {
       description: 'Directory where your files is being served from e.g. ./dist',
       default: null,
+    }).option('version-bump', {
+      description: 'Specify version bump type for to be published app',
+      default: 'major'
```

Comment:
I would introduce the `--version-bump` flag without a default value for now, requiring bumping on publish to be explicit.

We can still make that when `aragon run` uses the publish command as part of its execution, it always does a major bump as the old `automaticallyBump` did.

Reply:
https://github.com/aragon/aragon-cli/blob/6137e8f4987ddfcf7f85553641b3dd927badf367/src/commands/run.js#L168